iceoryx_posh/include/iceoryx_posh/internal/runtime/node_data.hpp

```diff
@@ -31,15 +32,15 @@ class NodeData
     /// @brief constructor
     /// @param[in] name name of the node
     /// @param[in] nodeDeviceIdentifier identifier of the device on which the node will run
-    NodeData(const ProcessName_t& process, const NodeName_t& node, const uint64_t nodeDeviceIdentifier) noexcept;
+    NodeData(const RuntimeName_t& process, const NodeName_t& node, const uint64_t nodeDeviceIdentifier) noexcept;
```

Comment:
Shouldn't we rename the ``process`` parameter as well? Please also add the ``@param[in]`` for it.